src/components/dashboard/sidebar/tests/SidebarCard.test.jsx

```diff
@@ -9,7 +9,6 @@ describe('<SidebarCard />', () => {
   const childrenText = 'Some text goes here';
   const defaultProps = {
     children: <>{childrenText}</>,
-    linkIsLocal: false,
```

Comment:
nit: unrelated to your changes here, but just noticed we're importing `SidebarCard` as `SideboardCard` on line 6. Should we change this to `SidebarCard` to match the actual component name throughout this file?

Reply:
yup!